arwen/contexts/runtime.go

```diff
@@ -833,6 +838,37 @@ func (context *runtimeContext) GetAllErrors() error {
 	return context.errors
 }
 
+// EnableGasTrace sets true to the flag variable that traces gas consumption
```

Comment:
init gas trace on InitState

Reply:
the gasTracing printed would only be from the last runtime

Reply:
GasTracing should be per transaction execution. As it is done right now, this is a memory leak. Even in mandos we need to print gasTrace after evey step.